server/server.js

```diff
@@ -6,6 +6,9 @@ let isDev = process.env.NODE_ENV !== 'production'
   Module imports
 \******************************************************************************/
 
+// Import variables from .env file.
+require('dotenv').config()
```

Comment:
If the .env-file should be missing, do we have a graceful fallback, or something that tells us that it's missing?

Reply:
dotenv just does nothing silently if .env isn't found. dotenv simply exists for dev environment things, and does not affect production or public dev environments.